invenio_base/app.py

```diff
@@ -194,33 +194,12 @@ def base_app(import_name, env_prefix='APP', instance_path=None,
 
     :param import_name: The name of the application package.
     :param env_prefix: Environment variable prefix.
```

Comment:
`env_prefix` is not unused (see https://www.quantifiedcode.com/app/project/gh:inveniosoftware:invenio-base/diff/a273e8fbe86c8a6b089fe1f2f3af7199395365df/3de36f3ab75caa32d49572b0c9bc15bffef64537/issues?groups=code_patterns%3A4Phzg6k7%3Af0)